riotbuild/Dockerfile

```diff
@@ -272,3 +296,20 @@ ENV RIOTBUILD_BRANCH $RIOTBUILD_BRANCH
 RUN echo "RIOTBUILD_VERSION=$RIOTBUILD_VERSION" > /etc/riotbuild
 RUN echo "RIOTBUILD_COMMIT=$RIOTBUILD_COMMIT" >> /etc/riotbuild
 RUN echo "RIOTBUILD_BRANCH=$RIOTBUILD_BRANCH" >> /etc/riotbuild
+
+# These depend on llvm-7-dev to be installed
+
+# The local prefix should be ${DOCKER_REGISTRY}, but variables are not supported
+# in this position by buildx, leading to errors like this:
+#
+# Error: buildx failed with: error: failed to solve: failed to solve with
+# frontend dockerfile.v0: failed to create LLB definition: failed to parse
+# stage name "${DOCKER_REGISTRY}/c2rust-built": invalid reference format:
+# repository name must be lowercase
+#
+# Until github supports this, the hardcoded "local" works well enough for the
+# tests, and needs to be replaced manually with "riot" when going live. Once
+# replaced and published, the ill-effect downstream users will experience is
+# that if they change their c2rust build, they'll have to adjust this line.
+# (For others, c2rust will be built but not used here).
+COPY --from=local/c2rust-built /usr/bin/c2rust /usr/bin/c2rust-refactor /usr/bin/c2rust-transpile /usr/bin/
```

Comment:
When using multistage COPY, FROM etc... you just need to redeclare the ARG see https://github.com/fjmolinas/riotdocker/blob/master/flashers/edbg.Dockerfile as an example

Reply:
Thanks, I'll give it a try with the next CI test run.

Reply:
Try it locally first otherwise!

Reply:
buildx is not in Debian yet, and I'm very careful about third party debs, but I'm running all tests I can without it here, and with the reference script and the ARG it's hopefully just one more run.

Reply:
Hm, even with the repeated ARG I get the same circular dependency error; any clues?

(Then again, going for a different workflow probably sidesteps the issue -- so maybe it's best to let this be and go for the other workflow right away).